atomic_reactor/plugins/post_push_to_pulp.py

```diff
@@ -127,7 +128,9 @@ def push_tar(self, filename, image_names=None, repo_prefix="redhat-"):
                 cmd = "set -o pipefail; {0} {1} | tar --delete {2} | gzip - > {3}".format(
                     unpacker, filename, ' '.join(remove_layers), outfile.name)
                 self.log.debug("running %s", cmd)
+                sys.stderr.flush()
                 subprocess.check_call(cmd, shell=True)
+                sys.stderr.flush()
```

Comment:
At this point, the subprocess has exited and so anything it wrote to stderr has definitely been flushed.